cmd/shield/main.go

```diff
@@ -3286,6 +3287,57 @@ tenants:
 
 	/* }}} */
 
+	case "op sql": /* {{{ */
+		if len(args) != 1 {
+			fail(2, "Usage: shield %s QUERY\n", command)
+		}
+
+		in := struct {
+			Query string `json:"q"`
+		}{
+			Query: args[0],
+		}
+		b, err := json.Marshal(&in)
+		bail(err)
+
+		code, response, err := c.Curl("POST", "/v2/expert/sql", string(b))
+		bail(err)
+
+		if code != 200 {
+			var out struct {
+				Error      string `json:"error"`
+				Diagnostic string `json:"diagnostic"`
+			}
+			err = json.Unmarshal([]byte(response), &out)
+			bail(err)
+
+			fmt.Fprintf(os.Stderr, "query `@C{%s}` failed:\n", in.Query)
+			fmt.Fprintf(os.Stderr, "@R{error: %s}\n", out.Error)
+			if out.Diagnostic != "" {
+				fmt.Fprintf(os.Stderr, "@R{       %s}\n", out.Diagnostic)
+			}
+			os.Exit(1)
+		}
+
+		var out struct {
+			Columns []string `json:"columns"`
+			Rows []map[string] interface{} `json:"rows"`
```

Comment:
Fix that space or somebody else's gofmt is going to get it later.

Reply:
go fmt'd ftw